apps/admin/Dockerfile

```
@@ -1,6 +1,6 @@
 # ---------- Stage 1: Prune monorepo ----------
 ARG TURBO_VERSION=2
-FROM --platform=linux/amd64 node:24-alpine AS builder
+FROM --platform=linux/amd64 node:24.14.1-alpine AS builder
```

Hadolint (2.14.0)

[warning] Do not use --platform flag with FROM

(DL3029)